easybuild/easyconfigs/__archive__/a/astropy/astropy-1.0.6-goolf-1.4.10-Python-2.7.3.eb

```diff
@@ -4,8 +4,8 @@ name = 'astropy'
 version = '1.0.6'
 
 homepage = 'http://www.astropy.org/'
-description = """The Astropy Project is a community effort to develop 
-a single core package for Astronomy in Python and foster interoperability 
+description = """The Astropy Project is a community effort to develop
+a single core package for Astronomy in Python and foster interoperability
```

Comment:
This will result in having `developa` rather than `develop a` in the output of `module help` I think, without adding a space in front of each continuation line?

Reply:
No. `"""` starts a string which is kept just like it is. Every newline and space is preserved